libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -241,6 +261,11 @@ export abstract class BaseInstallManager {
           this._rushConfiguration.tempShrinkwrapFilename,
           this._rushConfiguration.getCommittedShrinkwrapFilename(this.options.variant)
         );
+        // Copy (or delete) common\temp-split\pnpm-lock.yaml --> common\config\rush\split-workspace-pnpm-lock.yaml
```

Comment:
What is in `common/temp-split/pnpm-lock.yaml`?  In my prototype [this file was not created at all](https://github.com/octogonz/rush-split-workspace-prototype/tree/main/3-pnpm-rush/common/temp-split), since all the data was stored in the `<project>/pnpm-lock.yaml` files.

Reply:
I made the split workspace works for both cases: 1. `shared-workspace-lockfile=true`: `split-workspace-pnpm-lock.yaml`; 2. `shared-workspace-lockfile=false`: `<project>/pnpm-lock.yaml`